bin/init.bash

```diff
@@ -142,7 +142,7 @@ set_storage_class() {
             ;;
 
         upcloud)
-            storage_class=upcloud-block-storage
+            storage_class=standard
```

Comment:
So this will change in the defaults if it hasn't been overridden?
Then we should add an admin notice that the default storage class has changed so it doesn't come as a surprise for anyone that relies on the current default.
Or make a snippet that sets the previous default as an override if it previously wasn't set.

Reply:
Good point André :+1: Could we create a snippet that looks at the current storage and if it is not "standard" add a override?

Reply:
I'll write an admin notice about this.
I will also update the migration template to include reading the changelog/release notes.